src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsStartup.cs

```diff
@@ -77,57 +73,74 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         app.Map("/subdir", app =>
         {
-            WebAssemblyTestHelper.ServeCoopHeadersIfWebAssemblyThreadingEnabled(app);
-
-            if (!env.IsDevelopment())
+            app.Map("/reexecution", reexecutionApp =>
             {
-                app.UseExceptionHandler("/Error", createScopeForErrors: true);
-            }
-
-            app.UseRouting();
-            UseFakeAuthState(app);
-            app.UseAntiforgery();
+                reexecutionApp.UseStatusCodePagesWithReExecute("/not-found-reexecute", true);
```

Comment:
nit: Normally for booleans we use the `parameterName:` syntax to make it obvious what's for